docs/specs/markdown.yml

```diff
@@ -78,7 +78,7 @@ outputs:
   h4.json: |
     {
       "!title": null,
-      "!raw_title": null,
+      "raw_title": "",
```

Comment:
Does this conflict with https://github.com/dotnet/docfx/commit/e55823dd9777c157ca5e1282a20e3d6ccfc69cda, since `""` and `undefined` is treated differently for raw_title?

Reply:
no, rawTitle is different with title, based on current template implementation and v2 logic https://github.com/dotnet/docfx/blob/da48dd4c820f4b15dbcf4cc350ab68fabc5292b4/src/Microsoft.DocAsCode.Build.ConceptualDocuments/HtmlDocumentUtility.cs#L32